dockerfiles/Dockerfile-azkaban

```diff
@@ -0,0 +1,27 @@
+###
+# Instructions - run these commands from the directory where Dockerfile-azkaban is located
+#
+# Build command:
+# (Note - make sure at least 4G are available to your docker VM if not on Linux, Mac default is 2G)
+# (Also this takes 10-20 minutes. Would LOVE for a docker expert to advise how to improve speed :) )
+# docker build -m 4G -f Dockerfile-azkaban .
+#
+# Run command:
+# docker run -d -p 8081:8081 <YOUR BUILT CONTAINER HASH>
+#
+# Then you can view the solo-server at localhost:8081
+#
+
+FROM openjdk:8
+
+# download Azkaban from OS repo and checkout version 3.37.0
+RUN git clone https://github.com/azkaban/azkaban.git && cd azkaban && git reset --hard ce1e17b && echo "lol"
+
+# build and install Azkaban
+RUN cd azkaban && ./gradlew clean build installDist
```

Comment:
To get smaller image and easier ways to reuse you docker cache when you hack your Dockerfile, I suggest you to use docker multi-stage build. 

- One stage to provide your archive built. You will use this stage to tweak your build process.
- A second stage will get archive from 1rst stage and just prepare the production environment.  Here an example : https://github.com/mikefaille/play-scala-rest-api-example/blob/master/Dockerfile

By this way, your production image will not be load with build env. like source code and java build dependencies.

Reply:
(@reallocf just in case, sorry for my English; I need to practice. Thank you for this opportunity sir :)

Reply:
Thanks @mikefaille! I'll look more into the link you gave. This looks great :)